src/pages/settings/Wallet/ActivatePhysicalCardPage.tsx

```diff
@@ -56,8 +56,7 @@ function ActivatePhysicalCardPage({
     const [lastPressedDigit, setLastPressedDigit] = useState('');
 
     const domainCards = CardUtils.getDomainCards(cardList)[domain] ?? [];
-    const physicalCard = domainCards.find((card) => !card.nameValuePairs?.isVirtual);
-    const cardID = physicalCard?.cardID ?? 0;
+    const physicalCard = domainCards.find((card) => card?.state === CONST.EXPENSIFY_CARD.STATE.NOT_ACTIVATED);
```

Comment:
Only physical Card can be in NOT_ACTIVATED state. I was also wondering about, ff it's more readable to add isVirtual check, but decided to skip it since it's redundant. What do you think?

Reply:
Ok. Got it. Since NOT_ACTIVATED state is only for physical state, it is fine not to add a redundant check.